setting_up_firely_server/configuration/logsettings.rst

```diff
@@ -246,6 +246,26 @@ Seq server::
 * Change ``serverUrl`` to the URL of your Seq server
 * ``restrictedToMinimumLevel``: as described for `Console`_.
 
+Elasticsearch
+^^^^^^^^^^^^^
+
+`Elasticsearch <https://www.elastic.co/elasticsearch>`_ is a search engine based on the Lucene library. It provides a distributed, multitenant-capable full-text search engine with an HTTP web interface and schema-free JSON documents.
+
+For the ``Elasticsearch`` sink, you can also specify arguments. One of them is the nodeUris for your
+Elasticsearch server::
+
+		"WriteTo": [
+			{
+  				"Name": "Elasticsearch",
+				"Args": {
+					"nodeUris": "http://localhost:9200"
+				}
+			}
+
+* Change ``nodeUris`` to the URL of your Elasticsearch node
```

Comment:
Should we also maybe add information on the configuration for connectionGlobalHeaders here? Otherwise you cannot connect to elastic I think. Also good to point to the information on how this connection string can be obtained as described by Cheng in this ticket:https://firely.atlassian.net/browse/VONK-5887. The user needs the base64 encoding of the elastic password as described on this page: https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#_add_more_nodes

Reply:
It is only if you want to use Base authentication. it is not mentioned in the official docks. Official docks have no information regarding authentication. The only thing mentioned there was barely findable about the Bearer token and I was not able in short term to configure it.

The decision NOT to specify it was made as the customer's DevOps team could probably (most likely) know better how to configure ElasticSearch and integrate as they already investigated it and asked us.

Probably authentication is not required at all and only we had issues with that. (I shared with @cheng13231 information on how to disable authentication and httpS requirement)

Should I add it?

Reply:
Fair enough :) maybe good indeed then to add the information on disabling authentication, but I think it would still be good to mention that connectionGlobalHeaders can be added as a field when configuring authentication, no need to dive deep here.